src/components/styled.js

```diff
@@ -0,0 +1,65 @@
+import styled from "styled-components";
+
+export const Main = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  width: 100vw;
+  font-family: Verdana, Geneva, sans-serif;
+`;
+export const ProductionLineContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 100vw;
+  height: 100vh;
+  justify-content: space-around;
+  align-items: center;
+`;
+export const StationContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  background-color: lightblue;
+  min-height: 350px;
+  width: 300px;
+  border-radius: 10px;
+`;
+
+export const VehicleContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  min-height: 50px;
+  width: 80%;
+  border-radius: 10px;
+  justify-content: center;
+  align-items: center;
+  margin: 5px;
+  background: lightpink;
+`;
+
+export const StationLabel = styled.h2``;
+export const VehicleLabel = styled.h3``;
+
+export const StationStatus = styled.h6``;
+
+export const StationVehicle = styled.h6``;
```

Comment:
💡 To keep the tags semantically consistent, I created a few empty-styled components so I can give meaning to each tag I used.